assets/dev/js/editor/elements/views/behaviors/context-menu.js

```diff
@@ -1,4 +1,4 @@
-import AddSectionBase	from 'elementor-views/add-section/base';
+import AddSectionBase from 'elementor-views/add-section/base';
```

Comment:
not related to this PR

Reply:
I know, but I saw it, so I already fixed it. Shouldn't I?

Reply:
Ok, removed